tests/suites/deploy/deploy_charms.sh

```diff
@@ -153,8 +154,10 @@ run_deploy_local_lxd_profile_charm() {
 
 	ensure "test-deploy-local-lxd-profile" "${file}"
 
-	juju deploy ./testcharms/charms/lxd-profile
-	juju deploy ./testcharms/charms/lxd-profile-subordinate
+	# shellcheck disable=SC2046
+	juju deploy $(pack_charm ./testcharms/charms/lxd-profile)
+	# shellcheck disable=SC2046
+	juju deploy $(pack_charm ./testcharms/charms/lxd-profile-subordinate)
```

Comment:
I'd rather we quote properly than add shellcheck exceptions

Reply:
Quoting doesn't work because the output of `pack_charm` is a glob (e.g. `lxd-profile_*.charm`). If it is quoted, it will be treated like a literal file name, which is not what we want here.

Reply:
Ah I see